pkg/devfile/parser/context/apiVersion_test.go

```diff
@@ -9,8 +9,8 @@ import (
 func TestSetDevfileAPIVersion(t *testing.T) {
 
 	const (
-		apiVersion          = "1.0.0"
-		validJson           = `{"apiVersion": "1.0.0"}`
+		apiVersion          = "2.0.0"
+		validJson           = `{"apiVersion": "2.0.0"}`
```

Comment:
it is schema version for devfile v2, so valid json would be `{"schemaVersion": "2.0.0"}`